src/course-outline/drag-helper/DragContextProvider.tsx

```diff
@@ -0,0 +1,30 @@
+import { UniqueIdentifier } from '@dnd-kit/core';
+import React from 'react';
+
+interface DragContextProviderProps {
+  activeId?: UniqueIdentifier | null,
```

Comment:
Any reason to allow `null` AND `undefined` here?

I know this is based on previous code, so if you feel that fixing these `null`/`undefined` ambiguities will take a lot of time, feel free to skip it.